llvm/test/Transforms/MergeFunc/merge-fp-intrinsics.ll

```diff
@@ -0,0 +1,18 @@
+;; Make sure internal constrained FP intrinsics still merge properly
+; RUN: opt -passes=mergefunc -S < %s | FileCheck %s
+
+declare float @llvm.experimental.constrained.fadd.f32(float, float, metadata, metadata)
+
+define float @func1(float %a, float %b) {
+; CHECK-LABEL: define float @func1(float %a, float %b)
+  %result = call float @llvm.experimental.constrained.fadd.f32(float %a, float %b, metadata !"round.dynamic", metadata !"fpexcept.strict")
+  %result_2 = call float @llvm.experimental.constrained.fadd.f32(float %a, float %b, metadata !"round.dynamic", metadata !"fpexcept.strict")
+  ret float %result
+}
+
+define float @func2(float %a, float %b) {
+; CHECK-NOT: define float @func2(float %a, float %b)
```

Comment:
This doesn't do what you think it does: https://llvm.godbolt.org/z/xoqa7er8x The CHECK-NOT only passes because the arguments are renamed.

Please use update_test_checks.py instead. Always.